modules/architecture-overview.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='architecture-overview-{context}']
+= Architecture overview
+
+Like {product-title} v3, {product-title} v4 is a layered system designed to
+expose underlying container images and Kubernetes concepts as accurately as
+possible, with a focus on easy composition of applications by a developer.
```

Comment:
We need to touch this up, I’m not sure this is going to be the right way to phrase 4.  I think this is where we reset expectations for customers, and it’s going to be a combination of the core kubernetes mission (application infrastructure), the core 4 mission (automated and self-monitoring, flexible infrastructure), and the openshift story (giving developers tools to evolve their applications under operational oversight).